src/protocol.py

```diff
@@ -9,31 +19,32 @@
 import time
 import traceback
```

Comment:
It seems to be safe to sort these standard library imports. Not sure about the local imports below. Did you test?

Reply:
I tested that the gui ran but my test setup is lacking. As I said in the PR description, `Of interest for testing would be the change to imports. The app runs for me but my testing procedure is not thorough yet.`

Reply:
OK, at least my tests doesn't fail: https://travis-ci.org/g1itch/PyBitmessage/builds/383283944